### PR TITLE
MongoDB,Inc.MongoDBCLI version 1.21.0

### PR DESCRIPTION
**Pull request by gssbzn**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/40707)

### CODE REVIEW

File: manifests/m/MongoDB,Inc/MongoDBCLI/1.21.0/MongoDB,Inc.MongoDBCLI.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: MongoDB,Inc.MongoDBCLI
```
Comment (OfficialEsco) [inline manifests/m/MongoDB,Inc/MongoDBCLI/1.21.0/MongoDB,Inc.MongoDBCLI.installer.yaml:4]:
```suggestion
PackageIdentifier: MongoDB.CLI
```

File: manifests/m/MongoDB,Inc/MongoDBCLI/1.21.0/MongoDB,Inc.MongoDBCLI.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: MongoDB,Inc.MongoDBCLI
```
Comment (OfficialEsco) [inline manifests/m/MongoDB,Inc/MongoDBCLI/1.21.0/MongoDB,Inc.MongoDBCLI.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: MongoDB.CLI
```

File: manifests/m/MongoDB,Inc/MongoDBCLI/1.21.0/MongoDB,Inc.MongoDBCLI.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: MongoDB,Inc.MongoDBCLI
```
Comment (OfficialEsco) [inline manifests/m/MongoDB,Inc/MongoDBCLI/1.21.0/MongoDB,Inc.MongoDBCLI.yaml:4]:
```suggestion
PackageIdentifier: MongoDB.CLI
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/m/MongoDB,Inc/MongoDBCLI/1.21.0/MongoDB,Inc.MongoDBCLI.installer.yaml (hunk 1 above)
Comment (gssbzn) [reply manifests/m/MongoDB,Inc/MongoDBCLI/1.21.0/MongoDB,Inc.MongoDBCLI.installer.yaml]: Done

**Review by jedieaston — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**